cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -396,6 +396,18 @@ def set_sentry_client(self, sentry_url=None):
     def get_current_ip(self):
         return socket.gethostbyname(socket.gethostname())
 
+    def set_scws_realted_params(self):
+        # it is nowhere necessary within the dispatcher-app,
+        # but it is re-attached to the url within the email
+        # when sending it since it is used by the frontend
+        self.use_scws = self.par_dic.pop('use_scws', None)
```

Comment:
also if `use_scws` is set in parameters, but no actual scw list is given, issue should be raised.